cmd/mesh/manifest-apply.go

```diff
@@ -51,7 +52,7 @@ func addManifestApplyFlags(cmd *cobra.Command, args *manifestApplyArgs) {
 	cmd.PersistentFlags().StringVarP(&args.inFilename, "filename", "f", "", filenameFlagHelpStr)
 	cmd.PersistentFlags().StringVarP(&args.kubeConfigPath, "kubeconfig", "c", "", "Path to kube config")
 	cmd.PersistentFlags().StringVar(&args.context, "context", "", "The name of the kubeconfig context to use")
-	cmd.PersistentFlags().BoolVarP(&args.yes, "yes", "y", false, "Do not ask for confirmation")
+	cmd.PersistentFlags().BoolVar(&args.askForConfirm, "ask-confirm", true, "Whether ask for confirmation or not when manifest apply.")
```

Comment:
It's better to make the default be false if possible. In this case I suggest calling this "skip-confirmation" instead and make the description:
If set to true, the command will skip prompts and always assume a Yes response.

Reply:
@ostromart done 
cc @ayj